src/pages/Home.js

```diff
@@ -1,22 +1,38 @@
 import React from 'react';
+import { Link } from 'react-router-dom';
 import Button from '../components/Button';
-import Page from '../theme/Page';
-import Table from '../components/Table';
+import HeatMap from '../components/HeatMap/HeatMap';
+import Page from '../components/Page';
+import TextField from '../components/TextField';
+import RedditTopic from '../components/RedditTopic';
 
 
 export default function Home() {
   return (
     <Page>
-      <h1>
+      <TextField
+        align="center"
+        elem="h1"
+        mt="30"
+        mb="10"
+      >
         No reactions to your reddit posts?
-      </h1>
-      <div className="subtitle">
+      </TextField>
+      <TextField
+        align="center"
+        elem="p"
+        mt="10.5"
+        mb="45"
+      >
         Great timing, great results! Find the best time to post on your subreddit.
-      </div>
-      <Button
-        text="SHOW ME THE BEST TIME"
-      />
-      <Table />
+      </TextField>
+      <Link to="/search/javascript">
```

Comment:
Now it would make sense to extract the default subreddit `javascript` into a separate file. It's spread over three files already, the header, the reddit topic and the home page. Maybe even the heatmap if it's wrapped in a link.

If someone decided that `javascript` should be replaced by another default subreddit you would need to touch all these files and actually remember to update them.